Lib/asyncio/selector_events.py

```diff
@@ -618,19 +618,24 @@ def _sock_sendto(self, fut, sock, data, address):
         else:
             fut.set_result(n)
 
-    async def sock_connect(self, sock, address):
+    async def sock_connect(self, sock, address, proto_addr_info = None):
```

Comment:
Consistency with the rest of the file:
```suggestion
    async def sock_connect(self, sock, address, proto_addr_info=None):
```